compiler/luci/service/src/Nodes/CircleRsqrt.test.cpp

```diff
@@ -31,3 +32,68 @@ TEST(CloneNodeTest, clone_Rsqrt)
   auto cloned_rsqrt = dynamic_cast<luci::CircleRsqrt *>(cloned);
   ASSERT_NE(nullptr, cloned_rsqrt);
 }
+
+TEST(ShapeRuleTest, rsqrt_dynamic_shape)
+{
+  luci::CircleInput input;
+  luci::CircleRsqrt rsqrt;
+
+  loco::TensorShape shape;
+  luci::sinf::Rule shape_inf_rule;
+
+  input.shape({1, 1, 3, 4});
+  input.shape_status(luci::ShapeStatus::VALID);
+  input.dim(0).unset();
+  input.dim(1).unset();
+
+  rsqrt.x(&input);
+
+  ASSERT_TRUE(shape_inf_rule.infer(&rsqrt, shape));
+  ASSERT_EQ(shape.rank(), 4);
+  ASSERT_FALSE(shape.dim(0).known());
+  ASSERT_FALSE(shape.dim(1).known());
+  ASSERT_TRUE(shape.dim(2).known());
+  ASSERT_TRUE(shape.dim(3).known());
+
+  ASSERT_EQ(0, shape.dim(0).value());
+  ASSERT_EQ(0, shape.dim(1).value());
+  ASSERT_EQ(3, shape.dim(2).value());
+  ASSERT_EQ(4, shape.dim(3).value());
+}
+
+TEST(ShapeRuleTest, rsqrt_static_shape)
+{
+  luci::CircleInput input;
+  luci::CircleRsqrt rsqrt;
+
+  loco::TensorShape shape;
+  luci::sinf::Rule shape_inf_rule;
+
+  input.shape({3, 2, 3, 4});
+  input.shape_status(luci::ShapeStatus::VALID);
+
+  rsqrt.x(&input);
+
+  ASSERT_TRUE(shape_inf_rule.infer(&rsqrt, shape));
+  ASSERT_EQ(shape.rank(), 4);
+  ASSERT_TRUE(shape.dim(0).known());
+  ASSERT_TRUE(shape.dim(1).known());
+  ASSERT_TRUE(shape.dim(2).known());
+  ASSERT_TRUE(shape.dim(3).known());
+
+  ASSERT_EQ(3, shape.dim(0).value());
+  ASSERT_EQ(2, shape.dim(1).value());
+  ASSERT_EQ(3, shape.dim(2).value());
+  ASSERT_EQ(4, shape.dim(3).value());
+}
+
+TEST(ShapeRuleTest, rsqrt_shape_forward_NEG)
```

Comment:
How about renaming this TC name more explainable? 

```suggestion
TEST(ShapeRuleTest, rsqrt_void_input_NEG)
```
or 
```suggestion
TEST(ShapeRuleTest, rsqrt_invalid_input_NEG)
```

Reply:
That will be much more clear. I'll go with the second one. Thank you!